lib/commands/element.js

```diff
@@ -12,22 +12,26 @@ let commands = {}, extensions = {};
 // then override anything we want below
 Object.assign(extensions, iosCommands.element);
 
+commands.getNativeAttribute = async function getNativeAttribute (attribute, el) {
+  if (attribute === "contentSize") {
```

Comment:
single quotes?

Reply:
Sure. I'll fix when I rebase with the other changes from #830